colossalai/booster/plugin/gemini_plugin.py

```diff
@@ -349,6 +350,7 @@ def __init__(
         norm_type: float = 2.0,
         enable_tensor_parallelism: bool = False,
         tp_size: int = 1,
+        ddp_size:int = 1,
```

Comment:
Extra-dp-size may be clearer. By the way you shoud update docstring

Reply:
updated